buildsupport/jetty/pom.xml

```diff
@@ -89,66 +89,51 @@
 
       <dependency>
         <groupId>org.eclipse.jetty</groupId>
-        <artifactId>jetty-servlets</artifactId>
+        <artifactId>jetty-util-ajax</artifactId>
```

Comment:
order change here is just noise, would be better if it honored the previous order